### PR TITLE
Wilson/hotfix owner panel modal button

### PR DESCRIPTION
**Pull request by wilsonj806**
Fixes Owner Panel so that the modal button doesn't render like so:
bad modal button

It should be okay now but I'd prefer if other reviewers checked out hotfix branch and validate it

### CODE REVIEW

File: src/components/OwnerPanel/styles.module.scss [hunk 1]
```diff
@@ -75,7 +75,7 @@ $container-size: 160px;
     }
 
     @include for-tablet-landscape-up {
-      bottom: 0;
+      // bottom: -18px;
```
Comment (justintmckibben) [inline src/components/OwnerPanel/styles.module.scss:78]: ?

Comment (wilsonj806) [reply src/components/OwnerPanel/styles.module.scss]: Meant to delete that section. Amended!

**Comment by jacoblb64** [comment]
@wilsonj806 -- checked out your branch locally to test. Could you update seeds with some test data to test this, or let me know what data you populated your DB with?

**Comment by wilsonj806** [comment]
@jacoblb64 yup see below text file for some sample entries. I'll see if I can't get an updated seed file as well
[open_hours.txt](https://github.com/sendchinatownlove/sendchinatownlove.github.io/files/5467102/open_hours.txt)

**Review by jacoblb64 — APPROVED** [review]
Thanks for the seed info! Checked out locally and looks all good to me on mobile, tablet, and desktop! Was hoping to try out the delivery stuff, but as this PR isn't really about that I can't not approve 😉

**Comment by wilsonj806** [comment]
@jacoblb64 No problem, and I can share additional seed info for delivery/ delivery options on the slack for people to check it out!

**Pull request merged**